### PR TITLE
Update/5394 remove label printing mention for non us countries

### LINKED ISSUE #5394: Inbox notification - review shipping settings: remove label printing mention if the store country is not US
The label printing feature that WooCommerce Shipping offers only works in the United States. It will be necessary to create a new notification that targets non-US Stores without the mentioning label printing:
https://d.pr/i/5gIx5l

### PR DESCRIPTION
**Pull request by moon0326**
Fixes #5394 

This PR removes `Label printing: enabled` string from the shipping admin note for non-US countries by checking the store's base country.

Dev note: The code assumes that the WooCommerce Shipping extension and its label printing feature are enabled. I think it would be better if we actually perform a check on the WooCommerce Shipping extension availability and its settings, but I think that's a separate issue and requires a new PR.

### Screenshots

United States (notice we still have `Label printing: enabled`)
beforeth_

non-US countries:

afterth_

### Detailed test instructions:

It's tricky to test the change locally as we need Jetpack connection. I would recommend using http://jurassic.ninja/ to create a publicly accessible WordPress installation.

1. Create a fresh installation of WordPress.
2. Navigate to Plugins and install WooCommerce
3. Navigate to Plugins and click [ Add New]. Upload the attached zip file.
[woocommerce-admin.zip](https://github.com/woocommerce/woocommerce-admin/files/5484838/woocommerce-admin.zip)

4. Navigate to Plugins -> Installed Plugins and activate both WooCommerce and WooCommerce Admin plugin.
5. Click "WooCommerce" from the left sidebar to kick off the setup wizard.

**Scenario 1** - United States
**Expectation**: We should still see `Label printing: enabled` string from the admin note after finishing up the wizard.

Continuing from the above steps,

1. Fill out the address form and make sure to choose "United States (US) --- :state" from the Country/Region dropdown on the "Store Details" step and click [ Continue ]
2. Check at least one industry from the "Industry" step and click [ Continue ]
3. Choose "Physical products" from the "Product Types" step.
4. Continue and finish the wizard.
5. Make sure to choose "Yes please!" when "Enhance your storewith WooCommece Shipping & Tax" option pops up.
6. Make sure to connect to Jetpack and approve your site.

**Scenario 2** - Non-US countries
**Expectation**: We should not see `Label printing: enabled` string from the admin note after finishing up the wizard.

Same steps as scenario 1, except that you need to choose a country that is not one of United States, Australia, Canada, or United Kingdom. The reason we include Australia, Canada, and United Kingdom is that the shipping onboarding process won't continue for those countries according to https://github.com/woocommerce/woocommerce-admin/blob/main/src/Features/OnboardingSetUpShipping.php#L54

### CODE REVIEW

File: src/Notes/ReviewShippingSettings.php [hunk 1]
```diff
@@ -27,18 +27,23 @@ class ReviewShippingSettings {
 	public static function get_note() {
 		$note = new Note();
 
-		$note->set_name( self::NOTE_NAME );
-		$note->set_source( 'woocommerce-admin' );
-		$note->set_title( __( 'Review your shipping settings', 'woocommerce-admin' ) );
-		$note->set_content(
+		$note_lines = array(
```
Comment (becdetat) [inline src/Notes/ReviewShippingSettings.php:30]: Would `$content_lines` describe this better?

Comment (moon0326) [reply src/Notes/ReviewShippingSettings.php]: Good point! Yup, I think `$content_lines` makes more sense. I pushed a new commit.

**Comment by moon0326** [comment]
Thank you @becdetat. I made a new commit to reflect your suggestion 👍

**Comment by becdetat** [comment]
Looks good, merge when you're ready :-)

File: tests/features/class-onboarding-set-up-shipping.php [hunk 2]
```diff
@@ -66,4 +67,36 @@ public function test_zone_does_not_get_created_with_no_default_country() {
...
+	public function test_shipping_note_does_not_include_label_printing_for_non_us_countries() {
+		// Given AU:QLD as the default country.
+		update_option( 'woocommerce_default_country', 'AU:QLD' );
+
+		// When note is retrieved.
+		$note = ReviewShippingSettings::get_note()->get_content();
+
+		// Then it should not contain 'Label printing: enabled'.
+		$contains_string = strpos( $note, 'Label printing: enabled' );
+		$this->assertFalse( $contains_string );
```
Comment (jeffstieler) [inline tests/features/class-onboarding-set-up-shipping.php:84]: For future reference, there is an [`assertStringContainsString()`](https://phpunit.readthedocs.io/en/7.5/assertions.html#assertstringcontainsstring) assertion. 😄

**Comment by timmyc** [comment]
Great PR @moon0326 - thanks so much for adding test coverage, and being so detailed in your description too <3

**Comment by moon0326** [comment]
@timmyc Thank you :)

**Pull request merged**